packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -1969,7 +1970,9 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
 
 const styles = StyleSheet.create({
   verticallyInverted: {
-    transform: [{scaleY: -1}],
+    transform:
+      // It's important to invert the Y AND X axis to prevent a react native issue that can lead to ANRs on android 13
```

Comment:
Would be good to clarify that this is not a React Native issue, and make this comment reflect that we use 180deg rotation to avoid a bug in Android where scale + rotate transforms lead to ANRs. It might also make sense to call out that this is paired with a native change on Android that flips the scroll bar position to account for rotation.

Reply:
Bonus points if it links to the bug filed on Android so we know what to check if we ever want to consider cleaning this up.

Reply:
Adressed!